index.bs

```diff
@@ -3331,6 +3331,207 @@ be the same.
 </div>
 
 
+<h5 id="idl-overloading-vs-union">Overloading vs. union types</h5>
+
+<i>This section is informative.</i>
```

Comment:
There's some way of wrapping a section to say everything inside it is non-normative so that autolinks like `{{CanvasDrawPath}}` don't generate normative references, and so you get warnings when you try to use normative keywords. It appears to be undocumented. @tabatkins?

Reply:
Seems to be https://github.com/tabatkins/bikeshed/issues/1069.

Reply:
Yeah, if you put `class=non-normative` on an ancestor, the links will be non-normative. I don't currently look at headings, because it's an expensive check to do, but now that I'm memoizing I should see if it can be done relatively cheaply.